tasks/README.md

```diff
@@ -5,5 +5,7 @@ This directory contains the tasks that are part of this benchmark.
 
 Name | Summary | Category
 ---- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task249_CosE_IncorrectAns_generation` | Writing an IncorrectAns to a question(based on CosE v1.0). | Incorrect Answer Generation  
```

Comment:
Task 249: Please change to `Given a sentence and a question, generate an incorrect answer to the question.`
Task 250: Please change to `Given a question and an answer, generate an abstractive explanation for the question based on the answer.`
Task 251: Please change to `Given a paragraph, generate a question from the passage.`
Task 252: Please change to `Given a question, three answer options, and a sentence. Write the correct answer based on the sentence.`